.github/workflows/continuous_integration.yml

```diff
@@ -56,12 +56,6 @@ jobs:
       - name: "Install dependencies with composer"
         run: |
             composer update ${{ matrix.install-args }} --no-interaction --no-progress --prefer-dist
-        if: ${{ matrix.php-version != '8.0' }}
-
-      - name: "Install dependencies with composer. Ignoring platform reqs to bypass a problem with ecodev/graphql-upload available only with latest Webonyx on PHP8."
```

Comment:
PHP 8 support introduced in `ecodev/graphql-upload` [6.1.0](https://github.com/Ecodev/graphql-upload/releases/tag/6.1.0)
https://github.com/Ecodev/graphql-upload/commit/d602aa2cd421228ff5e080906f43dce32e93f4c3

This allows to drop workaround about skipping platform check (but also it is forbids to install `ecodev/graphql-upload` <6.1 for PHP 8 in `--prefer-lowest` mode)

Reply:
Can this also be removed on `master` now?

Reply:
You can just merge 5.0 branch to master after merging current PR.
This will make history clear and avoid missed commits (which is not a part of main branch).